tpcx_bb/cluster_configuration/cluster-startup-selene.sh

```diff
@@ -0,0 +1,109 @@
+#IB, NVLINK, or TCP
+ROLE=$1
+# CLUSTER_MODE="NVLINK"
+# CLUSTER_MODE="IB"
+CLUSTER_MODE="TCP"
+
+USERNAME=$(whoami)
+# HOSTNAME=$(hostname -i)
+HOSTNAME=$HOSTNAME
+
+MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
+DEVICE_MEMORY_LIMIT="32GB"
+POOL_SIZE="36GB"
+
+TPCX_BB_HOME=$HOME/tpcx-bb
+CONDA_ENV_NAME="rapids"
```

Comment:
Do you think it makes sense to generalize the env name across draco, selene, cluster_startup.sh to a common name rapids-tpcx-bb

Reply:
I think so. Will standardize on rapids-tpcx-bb